br/tests/br_incremental/run.sh

```diff
@@ -48,12 +56,17 @@ if [ "${row_count_full}" != "${row_count_ori_full}" ];then
 fi
 # incremental restore
 echo "incremental restore start..."
-run_br restore table --db $DB --table $TABLE -s "local://$TEST_DIR/$DB/inc" --pd $PD_ADDR
+run_br restore db --db $DB -s "local://$TEST_DIR/$DB/inc" --pd $PD_ADDR
 row_count_inc=$(run_sql "SELECT COUNT(*) FROM $DB.$TABLE;" | awk '/COUNT/{print $2}')
 # check full restore
 if [ "${row_count_inc}" != "${row_count_ori_inc}" ];then
     echo "TEST: [$TEST_NAME] incremental restore fail on database $DB"
     exit 1
 fi
 
+# check inserting records with auto increment id after incremental restore
+for i in $(seq $ROW_COUNT); do
```

Comment:
I doubt that 10 rows can cover this case.
maybe we should use some sql like `select max(_tidb_rowid) from a` to check tidb_rowid is expected after insertion.🤔